workflow/Snakefile

```diff
@@ -31,6 +31,23 @@ validate(config, Path("../config/config_schema.json"))
 
 
 OUT_DIR = Path(config["output_dir"])
+TMP_DIR = Path(config["tmp_dir"])
+LOG_DIR = Path(f"{config['local_log']}/../")
+
+
+###############################################################################
+### onSuccess/onError handlers configuration
+###############################################################################
+
+
+onsuccess:
+    print("\nWORKFLOW SUCCEED. Removing intermediate files.\n")
+    shell("rm -rf {TMP_DIR}")
+
+
+onerror:
+    print("\nWORKFLOW FAILED. Check the log file in the LOGS/ directory.\n")
```

Comment:
Is this the actual name of the directory, `LOGS/`? If not, just say "log directory", because the capitalization and trailing slash give the impression that it is indeed the name of the directory.

Reply:
It is in fact the `LOGS/` directory. I write it that way because it its in that directory whereas all the rule logs are in the `LOGS/LOCAL/` or `LOGS/CLUSTER` directory. Still, I can change it if you think that's better.

Reply:
If it always `LOGS/`, i.e., users can't change it through a config param, you can leave it. Otherwise it may be better to be more generic